pkg/kvstore/etcd.go

```diff
@@ -249,8 +249,10 @@ func Hint(err error) error {
 
 type etcdClient struct {
 	// firstSession is a channel that will be closed once the first session
-	// is set up in the etcd Client.
-	firstSession chan struct{}
+	// is set up in the etcd client. If an error occurred and the initial
```

Comment:
Typo in commit msg:

`established an thus the channel was leaked.`   

`established and thus the channel was leaked.`